docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[deploying-and-scaling]]
+== Deploying and Scaling Logstash
+
+The requirements for your Logstash installation change according to the scale demanded by your use case. In this 
+section, we will examine a range of Logstash installations in increasing order of complexity, starting from a minimal 
```

Comment:
I'm not a huge fan of the editorial "we". Perhaps, "This topic shows a variety of Logstash deployment scenarios, building from a minimal installation all the way through deploying multiple Logstash instances to ensure high availability."